openmc/universe.py

```diff
@@ -396,6 +400,18 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
                 height = pixels[0]*px/(params.top - params.bottom)
                 fig.set_size_inches(width, height)
 
+            if outline:
+                combined_rgb = np.sum(img, 2)
```

Comment:
```suggestion
                combined_rgb = np.sum(img, 2)
```

I just realized this only works most of the time. I shall replace this with something similar that works all the time